pages/system/notifications.js

```diff
@@ -40,182 +45,373 @@ export default class SystemPageNotifications extends React.Component {
         <br />
         <System.P>Import React and the Notification Component.</System.P>
         <br />
-        <br />
         <System.CodeBlock>
           {`import * as React from 'react';
 import { Notification } from 'slate-react-system';`}
         </System.CodeBlock>
         <br />
         <br />
-        <System.H2>Notification with status</System.H2>
+        <br />
+        <System.H2>Usage</System.H2>
         <hr />
         <br />
-        <System.Notification
-          label="Info Notification"
-          description="Here is the description"
-          status="INFO"
-        />
+        <System.P>
+          Declare the component at the root level of your document (e.g. in
+          index.js or App.js) so it is accessible throughout and will not get
+          buried in the DOM tree.
+        </System.P>
         <br />
-        <System.Notification
-          label="Success Notification"
-          description="Here is the description"
-          status="SUCCESS"
-        />
+        <System.P>
+          Use <System.CodeText>style</System.CodeText> to specify placement of
+          the fixed positioning notification list
+        </System.P>
+        <br />
+        <System.CodeBlock>
+          {`function App() {
+  return (
+    <React.Fragment>
+       <GlobalNotification style={{ bottom: 0, right: 0 }} />
+       {this.props.children}     
+    </React.Fragment>
+  )
+}`}
+        </System.CodeBlock>
+        <System.GlobalNotification style={{ bottom: 0, right: 0 }} />
         <br />
-        <System.Notification
-          label="Warning Notification"
-          description="Here is the description"
-          status="WARNING"
-        />
         <br />
-        <System.Notification
-          label="Error Notification"
-          description="Whoops that doesn't look good"
-          status="ERROR"
-        />
         <br />
+        <System.H2>Notification</System.H2>
+        <hr />
+        <br />
+        <System.ButtonSecondaryFull
+          onClick={() =>
+            this._createNotif({
+              id: this.state.count,
+              description: "This is a regular notification",
+            })
+          }
+        >
+          Click for notification
+        </System.ButtonSecondaryFull>
+        <br />
+        <System.ButtonSecondaryFull
+          onClick={() =>
+            this._createNotif({
+              id: this.state.count,
+              description: "This is a dark notification",
+              dark: true,
+            })
+          }
+        >
+          Click for dark style notification
+        </System.ButtonSecondaryFull>
+        <br />
+        <System.ButtonPrimaryFull
+          onClick={() => {
+            for (let i = 0; i <= this.state.count; i++) {
```

Comment:
I would prefer instead of a call to delete each notification you just had a way to assign a new empty array. Did you do it this way because you wanted to see each transition out?

Reply:
I assumed that people might want notifications that appeared at different times to disappear at different times, but I can definitely just assign an empty array if you think that's better. That would probably be easier too

Reply:
I think that is a lot better and simplifies the count.

Not sure why `this.state.count` is necessary, if its an Object you can do `Object.keys(object).length` or if its an array you can do `array.length`